frontend/app/routes/$lang/_public/status/myself.tsx

```diff
@@ -185,9 +183,8 @@ export default function StatusCheckerMyself() {
               {fetcher.data.status.name && <ClientFriendlyStatusMarkdown content={fetcher.data.status.name} />}
             </div>
           </ContextualAlert>
-          <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12">
+          <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={faChevronRight}>
```

Comment:
Any reason we aren't rendering the spinner icon when submitting?

```suggestion
          <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={isSubmitting ? faSpinner : faChevronRight} endIconProps={{ spin: isSubmitting }}>
```

Reply:
Because it's a button link and it doesn't submit any form.

Reply:
Missed that - thanks